transmogrifier/sources/xml/datacite.py

```diff
@@ -261,7 +261,7 @@ def get_alternate_titles(cls, source_record: Tag) -> list[timdex.AlternateTitle]
             ]
         )
         alternate_titles.extend(list(cls._get_additional_titles(source_record)))
-        return alternate_titles
+        return alternate_titles or None
```

Comment:
At this time, I do like this full encapsulation, thanks for making these changes.

Relevant here and elsewhere...

Once we get into orchestration, which I think touches on the instantiation of the `TIMDEXRecord` instance, it might be worth considering if maybe it's okay to return an empty list?  If so, and that was filtered out during serialization to JSON, then we could probably drop a bunch of these conversions to `None`.  Just noting for the future, but again, think this is great for now, and in the spirit of the incremental approach.

Reply:
I am most likely onboard with that approach when we get there!